### ISSUE TITLE
[09.04.24] EDC Update to 0.7.x_RC (24.05) Release

### ISSUE THREAD

**Issue by jzbmw** [eclipse-tractusx/item-relationship-service#358]
**As** Everyone ,
**I want** to make sure the IRS runs with the EDC Version which is necessary for R 24.05,
**so that** the whole network works .

## Link
- child of eclipse-tractusx/sig-release#446
- related to https://github.com/eclipse-tractusx/item-relationship-service/issues/424

## Blocked
- edc version does not exist yet. Wait until this version exists

## Hints / Details

- the EDC 0.7.0 Version will be available from the 05.04.24
- Hints: Wallet is no longer requested with technical user. Technical user is deprecated.
MIW URL is still in discussion
Dependencie to EDC 0.7.0 Version
- See https://github.com/eclipse-tractusx/item-relationship-service/issues/424 for further infos

````
please be informed that the BDRS URL to be used is:
[https://bpn-did-resolution-service.int.demo.catena-x.net/api/directory](https://bpn-did-resolution-service.int.demo.catena-x.net/api/directory/bpn-directory)
Example curl:
curl -X GET -IL  -k --fail https://bpn-did-resolution-service.int.demo.catena-x.net/api/directory/bpn-directory -H "content-type: application/json" -H "Authorization: Bearer <YOUR_VP_TOKEN>" -o /dev/null

A payload was successfully received using this approach.
Details also here:
https://confluence.catena-x.net/x/5ILrBw
````

### Acceptance Criteria
- [ ] Changes in EDC 0.7 are analyzed 
- [ ] The R24.05 released EDC Version is integrated
- [ ] An IRS Job has successfully finished with new EDC Version

#### MIW 
- [ ] Client_ID (not applicable anymore in EDC config)
- [ ]  Client_Secret (not applicable anymore in EDC config)
- [ ] Token_URL KeyCloak (not applicable anymore in EDC config)

### Required Changes 
- successfully deployed EDC 0.6.0 on DEV - providers, controlplanes and dataplanes
- /management/v2/assets API endpoints were removed and replaced with /management/v3/assets

Places to update:

1. reset-env.py
2. transform-and-upload.py -> request body changed from
```
{
"@context": {},
"asset": {
"@type": "Asset",
"@id": f"{asset_id_}",
"properties": {
"description": "IRS EDC Test Asset"
}
},
"dataAddress": {
"@type": "DataAddress",
"type": "HttpData",
"baseUrl": f"{submodel_url_}",
"proxyPath": "true",
"proxyBody": "false",
"proxyMethod": "false",
"proxyQueryParams": "false"
}
}

```
to
```
{
"@context": {},
"@id": f"{asset_id_}",
"properties": {
"description": "IRS EDC Test Asset"
},
"dataAddress": {
"@type": "DataAddress",
"type": "HttpData",
"baseUrl": f"{submodel_url_}",
"proxyPath": "true",
"proxyBody": "false",
"proxyMethod": "false",
"proxyQueryParams": "false"
}
}
```
3. IRS can no longer use v2 for creation of notification asset, confirm that Helm config value is set as below:
ess.assetsPath: /management/v3/assets  

## Out of Scope

- ...

**Comment by ds-jhartmann** [comment]
Currently blocked by missing instance of [bpn-did-resolution-service](https://github.com/eclipse-tractusx/bpn-did-resolution-service) and SAP DIM

**Comment by ds-jhartmann** [comment]
Updated EDC client to 0.7.0 in https://github.com/eclipse-tractusx/item-relationship-service/pull/586
testdata upload fix in https://github.com/eclipse-tractusx/item-relationship-service/pull/591
updated helm charts in https://github.com/catenax-ng/tx-item-relationship-service/pull/916

- [x] Changes in EDC 0.7 are analyzed
- [x] The R24.05 released EDC Version is integrated
- [ ] An IRS Job has successfully finished with new EDC Version

**Comment by ds-jhartmann** [comment]
Ready for additional Testing @mkanal @ds-kgassner

**Comment by mkanal** [comment]
PO acceptance in representation of @jzbmw . 
EDC version 0.7.0 is installed on DEV and INT integration tests works as expected.
